addons/website/static/src/builder/plugins/customize_website_plugin.js

```diff
@@ -192,9 +219,46 @@ export class CustomizeWebsitePlugin extends Plugin {
         await this.makeSCSSCusto(url, colors, nullValue);
     }, 0);
     async makeSCSSCusto(url, values, defaultValue = "null") {
-        Object.keys(values).forEach((key) => {
+        const old_values = {};
+
+        for (const key of Object.keys(values)) {
             values[key] = values[key] || defaultValue;
+            old_values[key] =
+                getComputedStyle(this.document.documentElement).getPropertyValue("--" + key) ||
```

Comment:
Is not there an util for that? `getSCSSVariable` or something like that (or was this removed? :thinking:)